src/parsestring.ts

```diff
@@ -5,6 +5,7 @@ import { Days } from './rrule'
 
 export function parseString(rfcString: string): Partial<Options> {
   const options = rfcString
+    .replace(/DTSTART:00,/, 'DTSTART:000')
```

Comment:
this seems a bit hackish too me. would not this be for trying to parse invalid values?